functions/pubsub/test/index.test.js

```diff
@@ -23,7 +23,7 @@ const {request} = require('gaxios');
 const PORT = 9020;
 const BASE_URL = `http://localhost:${PORT}`;
 
-const TOPIC = process.env.FUNCTIONS_TOPIC;
+const TOPIC = 'integration-tests-instance';
```

Comment:
small thought regrading process, it might be worth setting some of these environment varialbes, e..g, the reference to the `long-door-651` project in the actions files.